pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -308,15 +296,47 @@ func getPinnedLibraries(registry string) []libInfo {
 	return res
 }
 
-// getLibrariesLanguageDetection runs process language auto-detection and returns languages to inject for APM Instrumentation.
-// The languages information is available in workloadmeta-store and attached on the pod's owner.
-func (w *Webhook) getLibrariesLanguageDetection(pod *corev1.Pod) []libInfo {
-	if config.Datadog().GetBool("admission_controller.auto_instrumentation.inject_auto_detected_libraries") {
-		// Use libraries returned by language detection for APM Instrumentation
-		return w.getAutoDetectedLibraries(pod)
-	}
+type libInfoLanguageDetection struct {
+	libs    []libInfo
+	enabled bool
```

Comment:
IMHO, I think `enabled` can be misleading because when we read `libInfoLanguageDetection.enabled` we might think that this indicates if language detection is enabled or not, which is not the case. 

I assume this property refers to whether library injection is based on language detection or not, so maybe `injectionEnabled` might be clearer about the intent.